text/0000-io-target.md

```diff
@@ -0,0 +1,51 @@
+# SLIP-NNN - IO Target
+
+**By: Omid Bakhshandeh**
+This is a proposal for adding `scala.io.Target` to Scala std.
+
+## History
+
+| Date          | Version       |
+|---------------|---------------|
+| Jun 18th 2015 | Initial Draft |
+
+## Motivation
+
+`scala.io.Source` is one of the most used part of `Scala std`. For many people that are new
+to the language and programming IO play an important role.
+Scala std doesn't have any support for generating output. Scala programmers use `Java.io`
+or `Java.nio` for writing on disk. On the other hand, the incopatibility between Java File
+and Scala File makes it hard to have nice Scala code.
```

Comment:
Can you show a code snippet that uses Java nio to write to a file, and contrast with a snippet that uses this API?